lib/debug/session.rb

```diff
@@ -2293,11 +2293,19 @@ def skip?
   end
 
   def self.load_rc
-    [[File.expand_path('~/.rdbgrc'), true],
-     [File.expand_path('~/.rdbgrc.rb'), true],
-     # ['./.rdbgrc', true], # disable because of security concern
-     [CONFIG[:init_script], false],
-     ].each{|(path, rc)|
+    rc_file_paths = [
+      [File.expand_path('~/.rdbgrc'), true],
+      [File.expand_path('~/.rdbgrc.rb'), true],
+      # ['./.rdbgrc', true], # disable because of security concern
+    ]
+
+    if (xdg_home = ENV["XDG_CONFIG_HOME"])
+      rc_file_paths << [File.expand_path(File.join(xdg_home, "rdbg", "config")), true]
```

Comment:
don't you need `config.rb` here?

Reply:
Good catch. This change should support both `${XDG_CONFIG_HOME}/rdbg/config` and `${XDG_CONFIG_HOME}/rdbg/config.rb`. I'll make that update.

Reply:
@ko1 Thank you again for the feedback. I've added support for `config.rb` in this path in 55f6624.

Reply:
Thank you!